sessions/js-array-methods/spa/spa_01/css/styles.css

```diff
@@ -0,0 +1,89 @@
+*,
+*::before,
+*::after {
+	box-sizing: border-box;
+}
+
+:root {
+	--color-primary: #454551;
+}
+
+html {
+	height: 100%;
+	font-family: system-ui;
+}
+
+body {
+	display: grid;
+	grid-template-rows: 48px auto 48px;
+	position: fixed;
+	left: 0;
+	top: 0;
+	width: 100%;
+	height: 100%;
+	font-size: 112.5%;
+	margin: 0;
+}
+
+.header {
+	position: relative;
```

Comment:
I am very confused by this. At first glance, this doesn't do anything, and it's a very unusual choice for a header.